src/jws/flattened.rs

```diff
@@ -133,6 +141,8 @@ impl SignedData {
             signatures: (),
             unprotected_header: (),
         };
+        // This shouldn't fail, because FlattenedRaw strucs are
+        // always representable in JSON
         serde_json::to_string(&s).unwrap()
```

Comment:
Perhaps use an `expect` to give some meaningful error messages if it ever panics.

Reply:
Done